src/App.js

```diff
@@ -1,8 +1,31 @@
 import './App.scss';
+import React, {useEffect, useState} from "react";
+import Recipe from "./components/Recipe/Recipe";
+import {useRecipes} from "./context/RecipesContext";
+import AddRecipeForm from "./components/AddRecipeForm/AddRecipeForm";
+import EditRecipeForm from "./components/EditRecipeForm/EditRecipeForm";
 
 function App() {
-  return (
+    const {recipes, editMode, startEdit}  = useRecipes();
+    const [editRecipe, setEditRecipe] = useState(null);
+    const handleEdit = (recipe) => {
+        setEditRecipe(recipe);
+        startEdit();
+    }
+
+    return (
     <div className="App">
+        <AddRecipeForm />
+        {editMode ? <EditRecipeForm recipe={editRecipe}/> : null}
```

Comment:
If you want, you can use a simpler way to use the conditional expression:
{editMode && <EditRecipeForm recipe={editRecipe}/>}